sig-usability/README.md

```diff
@@ -0,0 +1,47 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Usability Committee
```

Comment:
nit: from the governance perspective, this isn't a committee. s/group/committee/g

Reply:
@jdumars I think this came from the generator?

Reply:
I think I found the problem, I put this in the wrong list, fixed.